plugins/console/index.js

```diff
@@ -76,14 +79,18 @@ ConsolePlugin.parseLog = function(context) {
       context.config.failOnWarning;
   var failOnError = (context.config.failOnError === undefined) ? true :
       context.config.failOnError;
+  var logWarnings = (context.config.logWarnings === undefined) ? true :
+      context.config.logWarnings;
   ConsolePlugin.exclude = context.config.exclude || [];
 
   return ConsolePlugin.getBrowserLog().then(function(log) {
-
-    var warnings = log.filter(function(node) {
-      return (node.level || {}).name === 'WARNING' &&
-          ConsolePlugin.includeLog(node.message);
-    });
+    var warnings = undefined;
```

Comment:
Better than making an edge case where `warnings` is `undefined`, why not use `var warnings = []`?  That way you won't have to add complexity on line 43